.github/workflows/basic.yaml

```diff
@@ -88,6 +93,10 @@ jobs:
       - name: Check wasm contracts
         run: cosmwasm-check ./target/wasm32-unknown-unknown/release/*.wasm
 
+      - name: Build ampd
```

Comment:
I'm thinking of putting this into a separate concurrent job. The shared cache isn't really useful since build target is different than for the contracts